src/styles/card.scss

```diff
@@ -0,0 +1,108 @@
+.card {
+  padding: 32px 16px 16px;
+  width: 200px; // Фиксированная ширина
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  background-color: #fff;
+  box-sizing: border-box;
+
+  &__imac {
+    padding: 0 3px;
+    width: 160px;
+    height: 134px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+  }
+
+  &__imac img {
+    width: 100%;
+    height: 100%;
+  }
+
+  &__title {
+    margin: 40px 0 0;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: #060b35;
+  }
+
+  &__subtitle {
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    padding: 4px 0 0;
+    margin: 0;
+    color: #616070;
+    text-align: left;
+  }
+
+  &__review-label {
+    margin: 0;
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    text-align: right;
+  }
+
+  &__review {
+    padding: 16px 0 0;
+    margin: 0;
+    display: flex; // Используем flex для выравнивания по горизонтали
+    align-items: center; // Выравниваем по центру
+    white-space: nowrap; // Предотвращаем перенос строк
+    gap: 17px;
+  }
+
+  &__button {
+    margin: 16px 0 0;
+    background-color: #00acdc;
+    color: #fff;
+    font-family: Roboto, sans-serif;
```

Comment:
Font-family should be defined in the root or body selector to avoid duplication and to keep the code DRY. If 'Roboto, sans-serif' is used throughout the project, consider setting it once at a higher level.